esmvaltool/cmorizers/obs/cmor_config/WOA.yml

```diff
@@ -1,43 +1,64 @@
 ---
 # Common global attributes for Cmorizer output
 attributes:
+
   dataset_id: WOA
-  version: '2013v2'
   tier: 2
   modeling_realm: clim
-  project_id: OBS
-  source: 'https://data.nodc.noaa.gov/woa/WOA13/DATAv2/'
-  reference: 'woa'
+
+  # WOA13
+  #project_id: OBS
+  #short_name: woa13
+  #version: '2013v2'
+  #source: 'https://www.ncei.noaa.gov/data/oceans/woa/WOA13/DATAv2/'
+  #reference: 'woa2013v2'
+
```

Comment:
Could you please add one extrea blank space before the attributes (e.g. # project_id: OBS)?